frontend/simple/views/containers/SignUp.vue

```diff
@@ -1,3 +1,4 @@
+<!-- TODO: Transform this view into a Modal, for layout consistence -->
```

Comment:
No longer doing this so this comment can be deleted

Reply:
Indeed. removed.